src/CC/ShellCheck/ShellScript.hs

```diff
@@ -20,9 +22,24 @@ import           System.FilePath.Posix
 
 --------------------------------------------------------------------------------
 
+-- | List of shells the engine should be able to handle.
+validShells :: [BS.ByteString]
+validShells = ["sh", "ash", "dash", "bash", "ksh"]
```

Comment:
I don't want to bikeshed over it too much, but I think it would be a good idea to ask them to expose it. 

I can create a placeholder issue on the clodeclimate-shellcheck repo which links to the ShellCheck issue, so if they _do_ expose `validShells` we have a reminder to use it - without preventing this code from being merged in the meantime.